forge-gui/res/cardsfolder/upcoming/brainstealer_dragon.txt

```diff
@@ -0,0 +1,16 @@
+Name:Brainstealer Dragon
+ManaCost:5 B B
+Types:Creature Dragon Horror
+PT:6/6
+K:Flying
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigExile | TriggerDescription$ At the beginning of your end step, exile the top card of each opponent's library. You may play those cards for as long as they remain exiled. If you cast a spell this way, you may spend mana as though it were mana of any color to cast it.
+SVar:TrigExile:DB$ Mill | Defined$ Player.Opponent | NumCards$ 1 | Destination$ Exile | RememberMilled$ True | SubAbility$ DBEffect
```

Comment:
Use Dig Effect instead of Mill

If there's another card that uses Mill effect to exile stuff, we should change that too :)

Reply:
The script for Mindleecher does use a Mill effect to exile cards from libraries this way, which served as a guide.

Good to know about the Dig alternative. I'm testing the swap now, will update the script once everything works

Reply:
Script updated:

- Replaced Mill effect with Dig effect
- Removed `TrigCleanup` trigger